pkg/odo/cli/application/application.go

```diff
@@ -137,3 +137,24 @@ func getMachineReadableFormat(client *occlient.Client, appName string, projectNa
 	}
 	return appDef
 }
+
+// checkApplicationName checks if the application is present or not
+func checkApplicationName(applicationName string) {
```

Comment:
Would it better to return an error here and let the caller decide on handling it?

Reply:
But then I will have to write the error handling code again and again for every call. I can rename to something like `checkApplicationNameAndError`. WDYT?

Reply:
@girishramnani is right, this is not a good pattern.

I would say that you don't even need this function.